pos_event_sale/models/pos_order_line.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Camptocamp SA - Iván Todorovich
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class PosOrderLine(models.Model):
+    _inherit = "pos.order.line"
+
+    event_ticket_id = fields.Many2one(
+        "event.event.ticket", string="Event Ticket", readonly=True
+    )
+    event_id = fields.Many2one(
+        "event.event",
+        string="Event",
+        related="event_ticket_id.event_id",
+        store=True,
+        readonly=True,
+    )
+    event_registration_ids = fields.One2many(
+        "event.registration",
+        "pos_order_line_id",
+        string="Event Registrations",
+        readonly=True,
+    )
+
+    @api.model
+    def create(self, vals):
+        res = super().create(vals)
+        res._create_event_registrations()
+        return res
+
+    def _create_event_registrations(self):
+        for line in self.filtered("event_ticket_id"):
+            ticket_id = line.event_ticket_id
+            vals = {
+                "event_ticket_id": ticket_id.id,
+                "event_id": ticket_id.event_id.id,
+                "partner_id": line.order_id.partner_id.id,
```

Comment:
If I want to buy 2 tickets for two people in the same order, I guess it will not possible because it seems you use the partner_id of the order_id. (or maybe I missed something, I don't use ``event`` module).

Reply:
That's correct. In fact the registration details are not handled currently by this module, as it wasn't in the scope of our project.
In my experience this is not needed most of the times, but I added a comment in the ROADMAP.